README.md

```diff
@@ -118,11 +119,18 @@ mirrord exec node app.js --target pod/my-pod
 
 ## How It Works
 
-When you select a pod to impersonate, mirrord launches a privileged pod on the same node as the pod you selected.
+When you select a pod to impersonate, mirrord launches a pod on the same node as the pod you selected.
 The new pod is then used to connect your local process and the impersonated pod: it mirrors incoming traffic from the pod to your process,
 routes outgoing traffic from your process through the pod, and does the same for file reads, file writes, and environment variables.
 You can read more about it [here](https://mirrord.dev/docs/overview/introduction/).
 
+### Additional capabilities
+
+Container run inside the pod launched by mirrord requires additional [Linux capabilities](https://man7.org/linux/man-pages/man7/capabilities.7.html):
```

Comment:
Great, this needs to be documented in mirrord.dev as well, with reference to mirrord for Teams (i.e if you don't want to have users allowed to create such pods, the operator can take care of it etc)